### ISSUE TITLE
Blocking Page: Messagebox doesn't clear the old message

### ISSUE THREAD

**Issue by Somebodyisnobody** [pi-hole/pi-hole#1912]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)). (Did my best to search!)

**How familiar are you with the the source code relevant to this issue?:**

2

---
**Expected behaviour:**

`Messagebox shows old messages + new messages (see screenshot)`

**Actual behaviour:**

![messagebox](https://user-images.githubusercontent.com/35230554/34910169-b448d474-f8af-11e7-85b5-4393adc1d699.png)

**Steps to reproduce:**

`Enter wrong password to call the error "Wrong password" -> Retype the password now correctly`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

no token

**Troubleshooting undertaken, and/or other relevant information:**

`I translated auth.php a bit and blockingpage.css full into german, but didn't touched the code`

**Comment by jacobsalmela** [comment]
The language might be a barrier, but I cannot reproduce this.  I entered the wrong password, received the error, entered the correct password, showed a success and then I was able to go to the Website.

Are you still seeing this issue?

<img width="1260" alt="screenshot 2018-02-17 21 53 46" src="https://user-images.githubusercontent.com/3843505/36348020-65b4e21e-142d-11e8-8c7c-14230c6ad7d8.png">
<img width="1257" alt="screenshot 2018-02-17 21 54 00" src="https://user-images.githubusercontent.com/3843505/36348021-65c68c80-142d-11e8-849f-6681c3fd1df0.png">
